src/com/sibvisions/kitchensink/samples/containers/SplitPanelSample.java

```diff
@@ -74,4 +119,20 @@ public String getName()
 		return "Split";
 	}
 	
+	public ICellEditor createOrientationCellEditor()
+	{
+		
+		UIEnumCellEditor cellEditor = new UIEnumCellEditor();
+		cellEditor.setAllowedValues(new Object[] {
+				new BigDecimal(ISplitPanel.SPLIT_LEFT_RIGHT),
+				new BigDecimal(ISplitPanel.SPLIT_TOP_BOTTOM)
+				
+		});
+		cellEditor.setDisplayValues(new String[] {
+				"Left/Right",
+				"Top/Bottom"
+		});
+		
+		return cellEditor;
+	}
 }	// SplitPanelSample

```

Comment:
Remove the empty line on the start of the function and instead add one here at the end of the class, before the closing brackets.